apps/crn-server/src/controllers/reminder.controller.ts

```diff
@@ -32,6 +35,25 @@ export default class ReminderController {
     return {
       total: reminders.total,
       items: reminders.items.map((reminder) => {
+        if (
+          reminder.entity === 'Research Output Version' &&
+          reminder.type === 'Published'
+        ) {
+          return {
+            id: reminder.id,
+            entity: reminder.entity,
+            href: sharedResearch({}).researchOutput({
+              researchOutputId: reminder.data.researchOutputId,
+            }).$,
+            description: `${capitalizeFirstLetter(
```

Comment:
Isn't this a bit repetitive ? "Team" A published a new "team" article etc. Same with working groups, I'd say it's worth discussing with Joao and Tiff